Library/Koala/KoalaTests.swift

```diff
@@ -659,4 +659,13 @@ final class KoalaTests: TestCase {
 
     XCTAssertEqual(["Changed Password"], client.events)
   }
+
+  func testTrackChangedCurrency() {
+    let client = MockTrackingClient()
+    let koala = Koala(client: client)
+
+    koala.trackChangedCurrency(.CAD)
+
+    XCTAssertEqual(["Selected Chosen Currency"], client.events)
```

Comment:
Is there a reason why are we not testing properties here?

```suggestion
    XCTAssertEqual(["Selected Chosen Currency"], client.events)
    XCTAssertEqual(Currency.CAD.descriptionText, client.properties.last?["currency"] as? String)
```

Reply:
Good call. I tested on the SettingsAccountViewModel, but missed here.